test/io/EventReader.test.cc

```diff
@@ -0,0 +1,104 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file EventReader.test.cc
+//---------------------------------------------------------------------------//
+#include "io/EventReader.hh"
+
+#include "gtest/Main.hh"
+#include "gtest/Test.hh"
+#include "base/Span.hh"
+#include "physics/base/ParticleParams.hh"
+
+using celeritas::EventReader;
+using celeritas::ParticleParams;
+using celeritas::Primary;
+namespace pdg = celeritas::pdg;
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+
+class EventReaderTest : public celeritas::Test,
+                        public testing::WithParamInterface<const char*>
+{
+  protected:
+    void SetUp() override
+    {
+        using celeritas::ParticleDef;
+        using celeritas::PDGNumber;
+
+        // Create shared standard model particle data
+        ParticleParams::VecAnnotatedDefs defs
+            = {{{"proton", pdg::proton()},
+                {938.27208816, 1, ParticleDef::stable_decay_constant()}},
+               {{"d_quark", PDGNumber(1)},
+                {4.7, -1 / 3, ParticleDef::stable_decay_constant()}},
+               {{"anti_u_quark", PDGNumber(-2)},
+                {2.2, -2 / 3, ParticleDef::stable_decay_constant()}},
+               {{"w_minus", PDGNumber(-24)}, {8.0379e4, 0, 3.168e24}},
```

Comment:
As vince pointed out, we should make sure to use units when defining these in unit tests: see https://github.com/celeritas-project/celeritas/pull/46/files#diff-51e18a1d57751a91864193c2a8bb590c

Reply:
Good point, I'll update accordingly.